src/eMall.Application/eMall.Application.csproj

```diff
@@ -15,4 +15,8 @@
     <ProjectReference Include="..\..\hommy-packages\Hommy.Result\Hommy.ResultModel.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Features\Products\Commands\" />
```

Comment:
Remove unnessesary items from cproj file

Reply:
It is expected behavior for empty folders.